File: twoSum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize) {
    int* mad=(int*)malloc(sizeof(int)*2);
    *returnSize=2;
    for(int i=0;i<numsSize;i++){
        for(int j=i+1;j<numsSize;j++)
        {
            if(nums[i]+nums[j]==target){
                mad[0]=i;
                mad[1]=j;
                return mad;
            }
        }
    }
    *returnSize=0;
    return NULL;
}

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer,Integer> n=new HashMap<>();
        int n1=nums.length;
        for(int i=0;i<n1;i++){
            int c=target-nums[i];
            if(n.containsKey(c)){
                return new int[] {n.get(c),i};

            }
            n.put(nums[i],i);
        }
        return new int[]{};
    }
}